olm_deploy/operatorregistry/registry-deployment.yaml

```diff
@@ -43,9 +43,14 @@ spec:
         command:
         - /usr/bin/registry-server
         - --database=/bundle/bundles.db
+        env:
+        - name: TMPDIR
+          value: /tmp
         volumeMounts:
         - name: workdir
           mountPath: /bundle
+        - name: tmpdir
+          mountPath: /tmp
```

Comment:
based on https://github.com/operator-framework/operator-registry/pull/332 its unclear to me how mounting `/tmp` resolves this

Reply:
https://github.com/operator-framework/operator-registry/commit/e9103486a9c8003f826eacc6b4357d7bd0a93c49#diff-1fbacfe86bbf565bbfefa5a9c465de06R12

Reply:
right, but wouldn't the pod already have a `/tmp` dir of its own? what does mounting it from the node accomplish?